src/proto/ctl/smd.proto

```diff
@@ -15,6 +15,8 @@ option go_package = "github.com/daos-stack/daos/src/control/common/proto/ctl";
 
 message BioHealthReq {
 	string dev_uuid = 1;
+	uint64 meta_size = 2;
```

Comment:
It's a bit awkward that this needs to be passed in to the request, but I guess that's the current design limitation. Could also do with having the description here.

Reply:
The explanation of why we have to pass it in the request is detailed in the description of the PR.
If you want, I could add some comments to explain why we have to introduce this new field.

Reply:
- [ ] Add comments